events/ri/src/main/resources/META-INF/services/org.projectnessie.events.spi.EventSubscriber

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (C) 2023 Dremio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Console printing event subscriber
+org.projectnessie.events.ri.console.PrintingEventSubscriber
+
+# Kafka event subscriber
+org.projectnessie.events.ri.kafka.KafkaEventSubscriber
```

Comment:
Is there a limit on the number of Subscribers that we can support?

Reply:
No hard limit, but the more subscribers, the more you mobilize Nessie's event loop and/or worker threads. I am planning some benchmarks later on.